src/common/watchdog/dud/watchdog.cpp

```diff
@@ -0,0 +1,27 @@
+// Copyright 2023 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+#include <common/watchdog.hpp>
+
+namespace mender {
+namespace common {
+namespace watchdog {
+
+void Kick() {
+	return;
+}
```

Comment:
I'd call this `noop` instead of `dud`, but not a biggie.

Reply:
Yeh - much better :smile: